include/multipass/ssh/sftp_dir_iterator.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2019-2022 Canonical, Ltd.
```

Comment:
Just 2022 please.

Reply:
Oops, sorry, I didn't notice the year differences, so I just copy-pasted these from whichever file I found first :sweat_smile: